### ISSUE TITLE
TypeError on MenuSurface

### ISSUE THREAD

**Issue by manuel3108** [hperrin/svelte-material-ui#79]
Recently I tried to put a MenuSurface in my Navigation bar. Apparently this doesn't seem possible at the moment. I always gut the following Error:
```js
TypeError: Cannot read property 'destroy' of undefined
    at eval (webpack:///./node_modules/@smui/menu-surface/MenuSurface.svelte?:62:16)
    at run (webpack:///./node_modules/svelte/internal/index.mjs?:160:12)
    at Array.forEach (<anonymous>)
    at run_all (webpack:///./node_modules/svelte/internal/index.mjs?:166:9)
    at Object.render (webpack:///./node_modules/svelte/internal/index.mjs?:1392:13)
    at handle_page (webpack:///./src/node_modules/@sapper/server.mjs?:2397:93)
```
But I wanted to know if I was mistaking or the component has a problem. So I placed it in my index.svelte. And after a rebuild, all in all it was working. But every time I refresh the page I could see an error displayed with the same content as described above.
So I had a look at [sveltematerialui.com](https://sveltematerialui.com/), especially on the demo page for the [MenuSurface](https://sveltematerialui.com/demo/menu-surface/). If you hard refresh this page (CTRL+F5) you can see an error 500 for a very short time.

For better understanding I created a minimal working reproduction at [manuel3108/smui-menu-surface-bug](https://github.com/manuel3108/smui-menu-surface-bug). You can see that this repository has 4 commits. First one is the default [sapper template](https://github.com/sveltejs/sapper-template) with one additional smui Button. In the second commit I added a simple long duration for loop, which is necessary for the next commit. Then in [55d5b72](https://github.com/manuel3108/smui-menu-surface-bug/commit/55d5b725ad6dd545b618abc3481f46badac972ad) i added the MenuSurface to the index.svelte. Here everything is working, even if its not looking good. You should now be able to clearly see the error/warning for a few seconds. [Last step is to relocate the MenuSurface in the Nav-Component](https://github.com/manuel3108/smui-menu-surface-bug/commit/09e8a297a45194b6cb2de2041e1caf6b65ea3036), which clearly breaks the application.

If this was not clear enough, checkout the repository and you will clearly understand what I mean :D

**Comment by damooo** [comment]
i too faced same problem with `MenuSurface` . it is reproducable. this error happening in service-worker build.

**Comment by eddiew** [comment]
Should be fixed in #80, pending merge. It happens because svelte calls onDestroy() on the server as well as the client.
